### PR TITLE
pashua -> gtkdialog

### PR DESCRIPTION
**Pull request by dericed**
This is a work-in-progress to replace Pashua with gtkdialog. Please install gtkdialog via `brew install amiaopensource/amiaos/gtkdialog` in order to test.

### CODE REVIEW

**Comment by dericed** [comment]
Examples of new UI
g
e

File: vrecord [hunk 1]
```diff
@@ -281,108 +173,267 @@ _get_decklink_inputs(){
...
+                <list selection-mode=\"1\" selected-row=\"$(_get_index_of_value "${PLAYBACKVIEW_CHOICE}" "${PLAYBACKVIEW_OPTIONS[@]}")\">
+                    <variable>PLAYBACKVIEW_CHOICE</variable>
+                    $(_expand_list2items "${PLAYBACKVIEW_OPTIONS[@]}")
+                </list>
+            </vbox>
+        </hbox>
+        <hbox>
+            <vbox>
+                <text>
+                    <label>Set recordig time (integer or decimal) in minutes</label>
```
Comment (kieranjol) [inline vrecord]: recording*

**Comment by kieranjol** [comment]
Looks lovely! So this should also work on linux as well?

**Comment by privatezero** [comment]
Can confirm this is working in linux:
image

**Comment by privatezero** [comment]
Since this appears viable, I have relatedly started going through for system specific variables over here:
https://github.com/amiaopensource/vrecord/tree/conditionals-for-system

**Comment by kieranjol** [comment]
> Since this appears viable, I have relatedly started going through for system specific variables over here:
> https://github.com/amiaopensource/vrecord/tree/conditionals-for-system

Awesome. Would the ffmpegdecklink homebrew formula also need something like this,as there's an osx-specific directory used there.

**Comment by retokromer** [comment]
@privatezero I added a comment on your branch.

**Comment by privatezero** [comment]
thanks @kieranjol - I've been hacking my way around ffmpeg-dl as I have been using the virtual input while on linux - I'll def look at that!

**Comment by privatezero** [comment]
thanks @kieranjol - I've been hacking my way around ffmpeg-dl as I have been using the virtual input while on linux - I'll def look at that!

File: vrecord [hunk 2]
```diff
@@ -21,9 +21,8 @@ unset TECHNICIAN
 RUNTYPE="record"
 CAPTURELOGSUFFIX="_ffmpeg_decklink_input.log"
 TIMECODELOGSUFFIX="_frame_timecodes.txt"
-unset PASHUAINSTALL
 unset EXTRAOUTPUTS
-UNDECLAREDOPTION="Undeclared"
+UNDECLAREDOPTION="unselected"
```
Comment (retokromer) [inline vrecord]: change the variable name as well?

**Comment by bturkus** [comment]
Woohoo! Thanks y’all

**Comment by dericed** [comment]
I'll merge soon if no objection

**Comment by privatezero** [comment]
no objections here - as far as I can test it in linux seems to be functional

**Comment by libbyhopfauf** [comment]
Hmmm, the GUI now consistently looks like this on all MIPoPS computers:
screen shot 2018-12-14 at 5 52 16 pm

**Comment by dericed** [comment]
@libbyhopfauf thx for reporting

**Comment by privatezero** [comment]
I suspect that the logos in `Resources` are not being installed via brew? Probably will need to add something in the manner of:
https://github.com/amiaopensource/homebrew-amiaos/blob/master/vrecord.rb#L29

as well as adapt the logos to use some variant of the logo path:
https://github.com/amiaopensource/vrecord/blob/master/vrecord#L13

**Pull request merged**